tests/driver/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+if (BUILD_ACQUIRE_DRIVER_ZARR)
+    set(NOTEST "TRUE")
+    add_subdirectory(${CMAKE_SOURCE_DIR}/acquire-common ${CMAKE_CURRENT_BINARY_DIR}/acquire-common)
+    set(NOTEST "FALSE")
+
+    #
+    # PARAMETERS
+    #
+    set(project acquire-driver-zarr) # CMAKE_PROJECT_NAME gets overridden if this is a subtree of another project
+
+    #
+    # Tests
+    #
+    set(tests
```

Comment:
Note: While I believe it's important to manually list all source and header files in the project's main build configuration, I prefer using `file(GLOB TEST_SOURCES ${CMAKE_CURRENT_LIST_DIR}/**/*.cpp)` for test-related builds. This dynamic file loading reduces overhead for test configurations.

This is just a suggestion—feel free to disregard. However, I'd be interested in your thoughts if you disagree.

Reply:
That's an interesting suggestion, but I think I'll save it for later experimentation.